README.md

```diff
@@ -105,12 +105,6 @@ The package is also available on [conda-forge](https://github.com/conda-forge/un
 conda install -c conda-forge unitpackage
 ```
 
-or mamba
-
-```sh .noeval
-mamba install -c conda-forge unitpackage
-```
-
```

Comment:
Do you remove it, because it is clear that conda can be replaced by mamba?

Reply:
`mamba` is not such a great deal anymore I think. `conda` fixed their solver (or rather, they use the mamba one). `mamba` still has some advantages but I think it's clear to the experts and to everyone else, `conda` is good enough.